lib/Differentiator/DiffPlanner.cpp

```diff
@@ -684,7 +685,7 @@ namespace clad {
       llvm::SaveAndRestore<const FunctionDecl*> saveTopMost = m_TopMostFD;
       m_TopMostFD = FD;
       TraverseDecl(derivedFD);
-      m_DiffPlans.push_back(std::move(request));
+      m_DiffRequestGraph.addNode(request, true /*isSource*/);
```

Comment:
```suggestion
      m_DiffRequestGraph.addNode(request, /*isSource=*/true);
```
There must be a clang-tidy check, why it did not warn here - can you check?

Reply:
Even after a lot of trials, I can't figure it out. I tried this option with multiple configurations of its options: https://clang.llvm.org/extra/clang-tidy/checks/bugprone/argument-comment.html.
Still doesn't work, changed it manually for now.